### ISSUE TITLE
Documented Return Type of CyberSource\Api\KeyGenerationApi:: generatePublicKey is incorrect

### ISSUE THREAD

**Issue by damienwebdev** [CyberSource/cybersource-rest-client-php#21]
# Bug

## Description

As a developer attempting to use the rest client, I expect the return types of functions to match the actual return types listed in doc blocks.

https://github.com/CyberSource/cybersource-rest-client-php/blob/7d623f9acbc526b9ee182fdf318e1c09321c7114/lib/Api/KeyGenerationApi.php#L99

```txt
Fatal error: Uncaught TypeError: Argument 1 passed to MyClass::transform() must be an instance of CyberSource\Model\FlexV1KeysPost200Response, array given
```

## Solution
Fix the return type to array, if it is actually an array, otherwise make the return type a `CyberSource\Model\FlexV1KeysPost200Response`

## Notes
This probably needs unit test coverage as well.

**Comment by gnongsie** [comment]
Hi,
I've provided the fix for this. A pull request will be sent shortly.

**Comment by gnongsie** [comment]
Hi,
The fix for this has been released as part of v0.0.6.

Thank you for bringing this to our notice. I will be closing this issue for now. 

Feel free to open a new issue for any further problems.
